packages/requirements.txt

```diff
@@ -1,2 +1,2 @@
-django==1.2 
+django==1.4.11 
```

Comment:
### :warning: Aqua detected vulnerability in your code  
**Vulnerability ID:** CVE-2015-0220   
**Check Name:** Django: Mitigated possible XSS attack via user-supplied redirect URLs   
**Severity:** MEDIUM   
**Fixed Version:** 1.4.18, 1.6.10, 1.7.3   
**Description:** The django.util.http.is_safe_url function in Django before 1.4.18, 1.6.x before 1.6.10, and 1.7.x before 1.7.3 does not properly handle leading whitespaces, which allows remote attackers to conduct cross-site scripting (XSS) attacks via a crafted URL, related to redirect URLs, as demonstrated by a "\njavascript:" URL.  
[This comment was created by Aqua Pipeline]  
  
Read more at https://avd.aquasec.com/nvd/cve-2015-0220